docs/web/docs/guides/how_to_use/worker_quality/using_golds.md

```diff
@@ -2,6 +2,8 @@
 sidebar_position: 4
 ---
 
+import Link from '@docusaurus/Link';
```

Comment:
Interesting - I didn't realize you could directly import react-style js into these markdowns and have them work. Good find!